### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

**Review by jpeach — APPROVED** [review]
Nit: Could make the commit message a full sentence :)

File: apis/v1alpha1/gateway_types.go [hunk 1]
```diff
@@ -566,7 +559,8 @@ type ListenerCondition struct {
 	// +required
 	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty" protobuf:"bytes,5,opt,name=lastTransitionTime"`
 	// If set, this represents the .metadata.generation that the condition was set based upon.
-	// For instance, if .metadata.generation is currently 12, but the .status.condition[x].observedGeneration is 9, the condition is out of date
+	// For instance, if .metadata.generation is currently 12,
+	// but the .status.conditions[x].observedGeneration is 9, the condition is out of date
```
Comment (robscott) [inline apis/v1alpha1/gateway_types.go:563]: Maybe just me, but the line length feels inconsistent here.

File: apis/v1alpha1/gatewayclass_types.go [hunk 2]
```diff
@@ -65,8 +65,7 @@ type GatewayClassSpec struct {
 
 	// AllowedGatewayNamespaceSelector is a selector of namespaces that Gateways
 	// can use this GatewayClass from. This is a standard Kubernetes
-	// LabelSelector, a label query over a set of resources. The result of
-	// matchLabels and matchExpressions are ANDed. Controllers must not support
+	// LabelSelector. Controllers must not support
```
Comment (robscott) [inline apis/v1alpha1/gatewayclass_types.go:68]: This results in inconsistent line length, maybe just wrap at 80 chars. If you happen to use VSCode, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) is awesome for automating this.

**Comment by hbagdi** [comment]
Addressed all comments, PTAL.

**Comment by robscott** [comment]
Thanks!

/lgtm

File: apis/v1alpha1/gateway_types.go [hunk 3]
```diff
@@ -97,11 +93,11 @@ type GatewaySpec struct {
 	// behavior can depend on the GatewayClass. If a value is set
 	// in the spec and the requested address is invalid, the
 	// GatewayClass MUST indicate this in the associated entry in
-	// GatewayStatus.Listeners.
+	// GatewayStatus.Addresses.
```
Comment (jpeach) [inline apis/v1alpha1/gateway_types.go:96]: This paragraph is trying to say that if an address can't be assigned, the controller should raise a `ConditionInvalidAddress`, but it's mangling it badly.

**Pull request merged**